src/olympia/devhub/tasks.py

```diff
@@ -110,9 +110,9 @@ def create_version_for_upload(addon, upload,
         # review since listed and sideload aren't supported for creation yet.
         if addon.status == amo.STATUS_NULL:
             if disallow_preliminary_review:
-                addon.update(status=amo.STATUS_PUBLIC)
+                addon.update(status=amo.STATUS_NOMINATED)
             else:
-                addon.update(status=amo.STATUS_LITE)
+                addon.update(status=amo.STATUS_PENDING)
```

Comment:
You're correct, it should be STATUS_UNREVIEWED.  I've not worked out why STATUS_LITE is somehow okay, but STATUS_PUBLIC is not.  Might be safer to leave it as STATUS_LITE if it's always worked this way?